spec/index.html

```diff
@@ -3775,6 +3788,150 @@ <h2>Acknowledgements</h2>
+    </li>
+    <li>
+      Babai, L., Erdős, P., Selkow, S.M.:
+      Random Graph Isomorphism.
+      SIAM J. Comput. 9, 628–635 (1980).
+      <a href="https://doi.org/10.1137/0209047">https://doi.org/10.1137/0209047</a>.
+    </li>
+    <li>
+      Hoffmann, C.M. ed:
+      Group-Theoretic Algorithms and Graph Isomorphism.
+      Springer, Berlin, Heidelberg (1982).
+      <a href="https://doi.org/10.1007/3-540-11493-9">https://doi.org/10.1007/3-540-11493-9</a>.
+    </li>
+    <li>
+      Corneil, D., Goldberg, M.:
+      A non-factorial algorithm for canonical numbering of a graph.
+      Journal of Algorithms. 5, 345–362 (1984).
+      <a href="https://doi.org/10.1016/0196-6774(84)90015-4">https://doi.org/10.1016/0196-6774(84)90015-4</a>.
+    </li>
+    <li>
+      Sutcliffe, G., Suttner, C.:
+      The TPTP Problem Library.
+      Journal of Automated Reasoning. 21, 177–203 (1998).
+      <a href="https://doi.org/10.1023/A:1005806324129">https://doi.org/10.1023/A:1005806324129</a>.
+    </li>
+    <li>
+      Grohe, M.:
+      Isomorphism testing for embeddable graphs through definability.
+      In: Proceedings of the thirty-second annual ACM symposium on Theory of computing. pp. 63–72.
+      ACM, Portland Oregon USA (2000).
+      <a href="https://doi.org/10.1145/335305.335313">https://doi.org/10.1145/335305.335313</a>.
+    </li>
+    <li>
+      Tsang, W.W., Hui, L.C.K., Chow, K.P., Chong, C.:
+      Tuning the collision test for stringency,
+      Tech. Rep. TR-2000-05, Computer Science and Information Systems,
+      Hong Kong University (2000).
+      <a href="https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=b9b3c7e4afd78e64c075f90694f0e6021960b3b1">https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=b9b3c7e4afd78e64c075f90694f0e6021960b3b1</a>
+    </li>
+    <li>
+      Berners-Lee, T., Connolly, D.:
+      [[[DesignIssues-Diff]]]
+      W3C Unofficial (2001).
+      <a href="https://www.w3.org/DesignIssues/Diff">https://www.w3.org/DesignIssues/Diff</a> [[DesignIssues-Diff]].
+    </li>
+    <li>
+      Carroll, J.J.:
+      [[[HPL-2003-142]]].
+      In: Fensel, D., Sycara, K., and Mylopoulos, J. (eds.)
+      The Semantic Web - ISWC 2003. pp. 369–384.
+      Springer, Berlin, Heidelberg (2003).
+      <a href="https://doi.org/10.1007/978-3-540-39718-2_24">https://doi.org/10.1007/978-3-540-39718-2_24</a> [[HPL-2003-142]].
+    </li>
+    <li>
+      Sayers, C., Karp, A.:
+      Computing the digest of an RDF graph,
+      Tech. Rep. HPL-2003-235 (R. 1),
+      Hewlett Packard Laboratories (2004).
+      <a href="https://www.hpl.hp.com/techreports/2003/HPL-2003-235R1.html">https://www.hpl.hp.com/techreports/2003/HPL-2003-235R1.html</a>.
+    </li>
+    <li>
+      Carroll, J.J., Bizer, C., Hayes, P., Stickler, P.:
+      Named graphs.
+      Journal of Web Semantics, 3 (4), pp. 247-267,
+      World Wide Web Conference 2005——Semantic Web Track  (2005).
```

Comment:
To reflect the above suggestion:
```suggestion
      Journal of Web Semantics. 3, 247–267 (2005).
```